web/src/settings_config.ts

```diff
@@ -537,12 +537,22 @@ export const expires_in_values = {
 const user_role_array = Object.values(user_role_values);
 export const user_role_map = new Map(user_role_array.map((role) => [role.code, role.description]));
 
+const enter_sends_label = `<span class="realm_enter_sends_kbd"><kbd>Enter</kbd> sends when composing a message</span>`;
+
 export const preferences_settings_labels = {
     dense_mode: $t({defaultMessage: "Dense mode"}),
     fluid_layout_width: $t({defaultMessage: "Use full width on wide screens"}),
     high_contrast_mode: $t({defaultMessage: "High contrast mode"}),
     starred_message_counts: $t({defaultMessage: "Show counts for starred messages"}),
     twenty_four_hour_time: $t({defaultMessage: "Time format"}),
+    realm_enter_sends: new Handlebars.SafeString(
+        $t_html(
+            {
+                defaultMessage: "<enter_sends_label></enter_sends_label>",
```

Comment:
This is nonsense; there's nothing to translate here. I think you might be best off just having a `setting_enter_sends_label.hbs` handlebars template and rendering that as the thing to pass to `new Handlebars.SafeString` here?